scripts/clean-merged-branches.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+#/ Usage: clean-merged-branches [-f]
+#/ Delete merged branches from the origin remote.
+#/
+#/ Options:
+#/   -f            Really delete the branches. Without this branches are shown
+#/                 but nothing is deleted.
+set -e
+
+# show usage maybe
+[ "$1" = "--help" ] && {
+    grep '^#/' <"$0"| cut -c4-
+    exit 0
+}
+
+# fetch and prune remote branches
+git fetch origin --prune
+
+# grab list of merged branches
+branches=$(
+  git branch -a --merged origin/master |
+  grep remotes/origin/ |
+  grep -v /master |
```

Comment:
This excludes `remotes/origin/Gongreg/master` as well

Reply:
I will fix that and add a confirm prompt instead of "run this command with `-f`" message